wolox-books/src/app/screens/sign-up/mocks/mock-user-service.ts

```diff
@@ -0,0 +1,25 @@
+import { User } from './../interfaces/user';
+import { Observable, of, throwError } from 'rxjs';
+
+export class MockUserService {
+
+  createUser(user: User): Observable<any> {
+    if (user.email === 'julian.lopera@wolox.co') {
+      return throwError({
+        errors: {
+          email: ['has already been taken'],
+          full_messages: ['Email has already been taken']
+        }
+      }
+    );
+    } else {
```

Comment:
Avoid use `else { return ...`
You can omit the `else` sentence.

Reply:
ok